Bottom: afe4315e808da2e97fd55c743608cb5aa0c45227
Top:    9b9bfbcd0d732c3144a506ec81bd44fd7a9822b1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 18:15:06 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
